apps/judicial-system/web/src/routes/Judge/Overview/Overview.tsx

```diff
@@ -226,10 +226,10 @@ export const JudgeOverview: React.FC = () => {
                   , verði með úrskurði Héraðsdóms Reykjavíkur gert að sæta
                   gæsluvarðhaldi til
                   <Text as="span" fontWeight="semiBold">
-                    {` ${formatDate(
-                      workingCase.requestedCourtDate,
-                      'EEEE',
-                    ).replace('dagur', 'dagsins')} 
+                    {` ${
+                      formatDate(workingCase.requestedCourtDate, 'EEEE') ||
+                      ''.replace('dagur', 'dagsins')
```

Comment:
Cool, I fixed this potential null pointer with a question mark 'formatDate(...)?.replace(...)'. Both approaches work.

Reply:
We could also let formatDate return '' instead of undefined.